github action: x86_64-linux-android build failed for cicada v0.9.23

### Problem description

In [this automatic commit](https://github.com/termux/termux-packages/commit/0d909a6de1d672bad429e5f008443a5d77a59983), the package cicada failed to build for target `x86_64-linux-android`. I want to provide some input here.

1. I have built this package successfully on a Linux box with Rust 1.58.1
2. On a Mac, I've cross-built it successfully for target `x86_64-linux-android` with NDK 22 (23 not working for `aarch64-linux-android`, so it's NDK 22 on my host).

So the root cause for this failure is not clear to me, but it shows we are using Rust 1.57 in Github Action. Maybe a newer version of Rust could resolve this?

Another direction is that I can update a dependency `libsqlite3-sys` to another sqlite lib. I think the complexity in this lib caused this issue.

Anyone have other inputs on this? Thanks.

<details>

  <summary>Outout of the Github Action:</summary>

```
...
Compiling terminfo v0.7.3
   Compiling mortal v0.2.2
   Compiling linefeed v0.6.0
error: linking with `x86_64-linux-android-clang` failed: exit status: 1
  |
  = note: "x86_64-linux-android-clang" "-Wl,--allow-multiple-definition" "-m64" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.0.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.1.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.10.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.11.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.12.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.13.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.14.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.15.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.2.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.3.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.4.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.5.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.6.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.7.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.8.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.cicada.26bb9efe-cgu.9.rcgu.o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc.3ot1l2grg8ookgdx.rcgu.o" "-Wl,--as-needed" "-L" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps" "-L" "/home/builder/.termux-build/cicada/src/target/release/deps" "-L" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/build/libsqlite3-sys-e0d406816de5c12a/out" "-L" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib" "-Wl,-Bstatic" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libuuid-e658d476ea392d00.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libserde-77443d777616748a.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libstructopt-fd389d2155400a2e.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libclap-5927d1b82642c7e0.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libtextwrap-86c3927877fb4aca.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libpest-87c8a321bd5862f4.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libucd_trie-24bd46c6e4aef5ae.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblazy_static-9df82b04e6313425.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libclap-657a4434334cf8bf.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libtextwrap-b496c676b72b538d.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libindexmap-6c6392beddd49c6d.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libos_str_bytes-b7495600acc7cc43.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libyaml_rust-df41a3f108a271b8.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblinked_hash_map-180d417d8a108a00.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libchrono-3c8677957f051589.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libnum_integer-2b9d2eb0dd44724d.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libnum_traits-f897d623e57c2a34.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libtime-3ded56609ac64a0f.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/librusqlite-c71a2c825fd27557.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libfallible_streaming_iterator-87995f9151027b95.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libfallible_iterator-2562bd657ca9aa7e.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libhashlink-253d67b6b9cc5c53.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libhashbrown-4ba3ecbdbc99784c.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libahash-254dd1dc0c80f225.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libgetrandom-9ff71f93d5d78602.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libonce_cell-6f5800620a92389b.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libregex-d9ef1db8f823ece9.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libaho_corasick-5be981bb27d585ae.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libregex_syntax-adc24b5d2209c6cc.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libnix-cc985de5dda646a0.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libmemoffset-7f0d8b6bc87ab8cd.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libcfg_if-2d3eadc3dc7e9d4c.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblinefeed-6acf3ee9129ab966.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libdirs-aee4c435adfb6762.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libmortal-7af4660b4cd636e0.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libterminfo-ffd9b5ee1fa7392d.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libdirs-595a2028ffd96373.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libdirs_sys-93c179b3f800a1cd.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libfnv-b41caad95bfea6bb.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libphf-e10eb4902eb1e166.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libphf_shared-ea21d349e8fa1397.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libsiphasher-03b4fc36f6a07e3e.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libnom-addfe31e3c4a22cb.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libmemchr-93a13f8621af1971.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libnix-9735a2dd5d9097db.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libvoid-f86454f5a029d3f5.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libcfg_if-59276dbaa0778e2b.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libunicode_width-f83444027c81ce98.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libunicode_normalization-8dd0d01f74cdc5e8.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libtinyvec-1c84f552b33d1663.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libtinyvec_macros-ab91b6a9678a7392.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libsmallstr-c9a7deddb6bd7758.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libsmallvec-c5a6df38445f550e.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libbitflags-84d990daf9f60e27.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libglob-66272610d2e9a40e.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/libexec-27dab9eb1edcda26.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liberrno-d675ca13b3f7efce.rlib" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibc-e3d3db497e9ebb1b.rlib" "-Wl,--start-group" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libstd-cfdca62ca0dd4bce.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libpanic_unwind-a95fc4406ba60256.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libminiz_oxide-b189ca955b7c2c1f.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libadler-b4f9d6cacf941297.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libobject-c43d0520744faa59.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libmemchr-2fc21f8d19b6f404.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libaddr2line-b2a2c6c5d33a0b79.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libgimli-78d18c0cb61455cf.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libstd_detect-bae7b467c766e2a9.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/librustc_demangle-f1ddc79cfc731319.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libhashbrown-3f637e56393c8c1c.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/librustc_std_workspace_alloc-6f5de594230aca85.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libunwind-9fa3166466e77caf.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libcfg_if-5aafb15a03afa2f1.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/liblibc-b50421ec0a376c5b.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/liballoc-7d5085ffaf0767cc.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/librustc_std_workspace_core-9498e126972b4584.rlib" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libcore-8221c5b8891dc3d6.rlib" "-Wl,--end-group" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib/libcompiler_builtins-5926075c9f4d3cf8.rlib" "-Wl,-Bdynamic" "-ldl" "-llog" "-lgcc" "-lc" "-lm" "-Wl,--eh-frame-hdr" "-Wl,-znoexecstack" "-L" "/home/builder/.rustup/toolchains/1.57.0-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-linux-android/lib" "-o" "/home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/cicada-01fda0c096d938fc" "-Wl,--gc-sections" "-pie" "-Wl,-zrelro" "-Wl,-znow" "-Wl,-O1" "-nodefaultlibs" "-Wl,-rpath=/data/data/com.termux/files/usr/lib" "-Wl,--enable-new-dtags"
  = note: ld: error: undefined symbol: __extenddftf2
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced 2 more times
          
          ld: error: undefined symbol: __trunctfdf2
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced 3 more times
          
          ld: error: undefined symbol: __multf3
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced 9 more times
          
          ld: error: undefined symbol: __addtf3
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          
          ld: error: undefined symbol: __gttf2
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3IntFloatCompare) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          
          ld: error: undefined symbol: __getf2
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced 1 more times
          
          ld: error: undefined symbol: __divtf3
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3AtoF) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3AtoF) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          
          ld: error: undefined symbol: __lttf2
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3_str_vappendf) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3IntFloatCompare) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          
          ld: error: undefined symbol: __fixtfsi
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(et_getdigit) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          
          ld: error: undefined symbol: __floatsitf
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(et_getdigit) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          
          ld: error: undefined symbol: __subtf3
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(et_getdigit) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          
          ld: error: undefined symbol: __floatditf
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3AtoF) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3AtoF) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          >>> referenced by sqlite3.c
          >>>               sqlite3.o:(sqlite3IntFloatCompare) in archive /home/builder/.termux-build/cicada/src/target/x86_64-linux-android/release/deps/liblibsqlite3_sys-aed2f33501011958.rlib
          clang-12: error: linker command failed with exit code 1 (use -v to see invocation)
          

error: failed to compile `cicada v0.9.23 (/home/builder/.termux-build/cicada/src)`, intermediate artifacts can be found at `/home/builder/.termux-build/cicada/src/target`

Caused by:
  could not compile `cicada` due to previous error
Error: Process completed with exit code 101.
```

</details>

### What steps will reproduce the bug?

n/a

### What is the expected behavior?

n/a

### System information

n/a

--- a/packages/cicada/build.sh
+++ b/packages/cicada/build.sh
@@ -3,6 +3,7 @@ TERMUX_PKG_DESCRIPTION="A bash like Unix shell"
 TERMUX_PKG_MAINTAINER="Hugo Wang <w@mitnk.com>"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_VERSION=0.9.23
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://github.com/mitnk/cicada/archive/v$TERMUX_PKG_VERSION.tar.gz
 TERMUX_PKG_SHA256=fafc780b78153bbbb7160a2ec1354dde313fa380df86593bb806b69602bce145
 TERMUX_PKG_AUTO_UPDATE=true
@@ -10,5 +11,14 @@ TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_BLACKLISTED_ARCHES="arm, i686"
 
 termux_step_pre_configure() {
-    rm -f Makefile
+	rm -f Makefile
+
+	if [ "$TERMUX_ARCH" == "x86_64" ]; then
+		local libdir=target/x86_64-linux-android/release/deps
+		mkdir -p $libdir
+		pushd $libdir
+		local libgcc="$($CC -print-libgcc-file-name)"
+		echo "INPUT($libgcc -l:libunwind.a)" > libgcc.so
+		popd
+	fi
 }

